### PR TITLE
Studies in user table

### PR DESCRIPTION
**Pull request by Nunien**
* study list column in user table - https://github.com/fli-iam/shanoir-ng/issues/1084
* fix bug : in another user details, click on profile -> no change
* study-user list separate component
* change study list style in user

### CODE REVIEW

**Comment by jcomedouteau** [comment]
@Nunien Theres (already ?) conflict to resolve !
Can you describe this bug -> fix bug : in another user details, click on profile -> no change ?
I don't understand it well.
Thanks!

File: shanoir-ng-front/src/app/studies/shared/study-user.model.ts [hunk 1]
```diff
@@ -32,11 +33,7 @@ export class StudyUser {
...
 
     public static completeMember(studyUser: StudyUser, users: User[]) {
-        for (let user of users) {
-            if (studyUser.userId == user.id) {
-                studyUser.user = user;
-                user.selected = true;
-            }
-        }
+        let user: User = users.find(u => u.id == studyUser.userId);
+        if (user) studyUser.user = user;
```
Comment (jcomedouteau) [inline shanoir-ng-front/src/app/studies/shared/study-user.model.ts:37]: The user.selected = true is not usefull anymore ?

**Comment by jcomedouteau** [comment]
J'ai eu des soucis avec le nouveau composant study-user:
1) Mon utilisateur jlouis apparait comme faisant parti de l'étude ET dans les options d'ajout (ce qui n'est pas le cas des autres utilisateurs de l'étude)
image
Si j'ajoute de nouveaux utilisateurs, ils ne disparaissent pas de la liste.

2) Si je supprime un utilisateur existant (ici mkain), j'ai une erreur et l'utilisateur n’apparaît pas dans les options:
image

Merci !

**Comment by Nunien** [comment]
@jcomedouteau Sorry I can't reproduce that. I suspect there might be a problem with your users data. All the user users should apprear in the select box and the selected ones should be greyed. Is the response from https://shanoir-ng-nginx/shanoir-ng/users/users [GET] normal ? Does every user have an id ?

**Comment by jcomedouteau** [comment]
I'm re-testing !

**Comment by jcomedouteau** [comment]
You are right, my users are not well initialized in keycloak. i'm updating this.

**Comment by jcomedouteau** [comment]
I just had the little problem that when you re-edit the study, all available users (even if they already are in the study) are displayed as "addable". It does nothing when clicking on "add", but they are not grey.
image

**Comment by Nunien** [comment]
> 

Yes, I think an issue should be created as the problem existed before and it would require a new api service or a rabbitmq transaction to deal with it.

**Comment by Nunien** [comment]
> It's kind of weird, when deleting an user, we don't delete associated study_user object -> This should be done no ?

Yes, I think an issue should be created as the problem existed before and it would require a new api service or a rabbitmq transaction to deal with it.

**Comment by Nunien** [comment]
> I just had the little problem that when you re-edit the study, all available users (even if they already are in the study) are displayed as "addable". It does nothing when clicking on "add", but they are not grey.

I can't reproduce that

**Review by jcomedouteau — APPROVED** [review]
Ok could not reproduce on qualif my last bug. I'm creating a new issue for the other one.

**Pull request merged**